components/director/internal/model/application_test.go

```diff
@@ -66,3 +71,44 @@ func TestApplicationCreateInput_ToApplication(t *testing.T) {
 		})
 	}
 }
+
+func TestApplicationUpdateInput_UpdateApplication(t *testing.T) {
```

Comment:
What does this test check? I believe it is not necessary.

Reply:
I would remove `"Only needed properties"`, because it doesn't make any sens.